### PR TITLE
PR to see changes in v2023.06.0-RC branch vs. develop.

### PR DESCRIPTION
**Pull request by rhornung67**
# Summary

- This PR makes it easier to see the changes between the v2023.06.0 release candidate branch and the develop branch.
- Please review this PR and approve the PR to merge this branch into main linked here: https://github.com/LLNL/RAJA/pull/1502

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -19,6 +19,63 @@ Notable changes include:
...
+       were appropriate. In this release, seq_exec behaves the same as how
+       loop_exec behaved historically. The loop_exec and associated policies
+       such as loop_atomic, loop_reduce, etc. are type aliases to the 
+       analogous seq_exec policies. This prevents breaking user code with this
+       release. However, users should prepare to switch loop_exec policies 
+       to the seq_exec policy variants in the future.
+     * GPU global (thread and block) indexing has been refactored to abstract 
+       indexing in a given dimension. The result is that users can now specify 
+       a block size or a grid size at compile time or get those values at 
+       run time. You can also ignore blocks and index only with threads and            vice versa. Kernel and launch policies are now shared. Such policies are
```
Comment (MrBurmark) [inline RELEASE_NOTES.md:52]: spacing

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -19,6 +19,63 @@ Notable changes include:
...
+       now multi-part and contain global indexing information, a way to map 
+       global indices like direct or strided loops, and have a synchronization 
+       requirement. The synchronization allows one to request that all threads 
+       complete the even if some have no work so you can synchronize a block.
```
Comment (MrBurmark) [inline RELEASE_NOTES.md:56]:
```suggestion
       complete even if some have no work so you can synchronize a block.
```

File: RELEASE_NOTES.md [hunk 3]
```diff
@@ -19,6 +19,63 @@ Notable changes include:
...
+       Aliases have been added for all of the pre-existing policies and some 
+       are deprecated in favor more consistently named policies. One
```
Comment (MrBurmark) [inline RELEASE_NOTES.md:58]:
```suggestion
       are deprecated in favor of more consistently named policies. One 
```

File: RELEASE_NOTES.md [hunk 4]
```diff
@@ -19,6 +19,63 @@ Notable changes include:
...
+       Aliases have been added for all of the pre-existing policies and some 
+       are deprecated in favor more consistently named policies. One 
+       **BREAKING CHANGE** is that thread loop policies are no longer safe to 
+       block synchronization. That feature still exists but can only be 
+       accessed with a custom policy. The RAJA User Guide contains descriptions
+       of the new policy mechanics. 
+
+  * Build changes/improvements:
+     * Update BLT submodule to v0.5.3.
+     * Umpdate camp submodule to v2023.06.0
```
Comment (MrBurmark) [inline RELEASE_NOTES.md:66]:
```suggestion
     * Update camp submodule to v2023.06.0
```

File: RELEASE_NOTES.md (hunk 1 above)
Comment (rhornung67) [reply RELEASE_NOTES.md:52]: Fixed

File: RELEASE_NOTES.md (hunk 2 above)
Comment (rhornung67) [reply RELEASE_NOTES.md:56]: Fixed

File: RELEASE_NOTES.md (hunk 3 above)
Comment (rhornung67) [reply RELEASE_NOTES.md:58]: Fixed

File: RELEASE_NOTES.md (hunk 4 above)
Comment (rhornung67) [reply RELEASE_NOTES.md:66]: Fixed

File: RELEASE_NOTES.md [hunk 5]
```diff
@@ -19,6 +19,64 @@ Notable changes include:
...
+       time. You can also ignore blocks and index only with threads and
+       vice versa. Kernel and launch policies are now shared. Such policies are
+       now multi-part and contain global indexing information, a way to map
+       global indices like direct or strided loops, and have a synchronization
+       requirement. The synchronization allows one to request that all threads
+       complete even if some have no work so you can synchronize a block.
+       Aliases have been added for all of the pre-existing policies and some
+       are deprecated in favor of policies named more consistently. One
+       **BREAKING CHANGE** is that thread loop policies are no longer safe to
+       block synchronization. That feature still exists but can only be
```
Comment (MrBurmark) [inline RELEASE_NOTES.md:61]:
```suggestion
       block synchronize. That feature still exists but can only be
```

Comment (rhornung67) [reply RELEASE_NOTES.md]: Changed.

**Pull request closed without merging**